packages/common/src/data/App/index.json

```diff
@@ -4,19 +4,19 @@
       "id": 1,
       "icon": "flaticon-atom",
       "title": "Discover processes",
-      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. "
+      "description": "Browse the processes & find all tools you need to get started and elevate your operations."
```

Comment:
Browse industry specific processes & tools that optimize your self-management workflow.